### ISSUE TITLE
Video file detected as audio

### ISSUE THREAD

**Issue by 4l4R1** [CasparCG/media-scanner#76]
Hei!

I found a video file (mpeg2 video in an MPEG-PS container) that Scanner v1.3.3 recognized as an audio file

`"TESTVIDEO"  AUDIO  61941764 20210823173547 872640 1/90000`

However, there are files from the same time that are all correctly detected.

`"TESTVIDEO2"  MOVIE  76900356 20210824111057 301 1/25`

In Mediainfo the only significant difference I found was that
there was a line in the "audio file" that was missing from the "video file:
`Format settings, GOP : M=3, N=15`

If it helps I can send the files.

Mediainfos:

```
Complete name               : E:\Caspar\Server\media\testvideo.mpg
Format                      : MPEG-PS
File size                   : 59.1 MiB
Duration                    : 9 s 696 ms
Overall bit rate mode       : Constant
Overall bit rate            : 51.1 Mb/s
Frame rate                  : 25.000 FPS

Video
ID                          : 224 (0xE0)
Format                      : MPEG Video
Format version              : Version 2
Format profile              : Main@High
Format settings             : BVOP
Format settings, BVOP       : Yes
Format settings, Matrix     : Default
Format settings, GOP        : M=3, N=15
Format settings, picture st : Frame
Duration                    : 9 s 640 ms
Bit rate mode               : Constant
Bit rate                    : 50.0 Mb/s
Width                       : 1 920 pixels
Height                      : 1 080 pixels
Display aspect ratio        : 16:9
Frame rate                  : 25.000 FPS
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Interlaced
Scan order                  : Top Field First
Compression mode            : Lossy
Bits/(Pixel*Frame)          : 0.965
Time code of first frame    : 00:00:00:00
Time code source            : Group of pictures header
Stream size                 : 57.1 MiB (97%)

Audio
ID                          : 192 (0xC0)
Format                      : MPEG Audio
Format version              : Version 1
Format profile              : Layer 2
Duration                    : 9 s 696 ms
Bit rate mode               : Constant
Bit rate                    : 384 kb/s
Channel(s)                  : 2 channels
Sampling rate               : 48.0 kHz
Frame rate                  : 41.667 FPS (1152 SPF)
Compression mode            : Lossy
Stream size                 : 455 KiB (1%)
```

```
Complete name               : E:\Caspar\Server\media\testvideo2.mpg
Format                      : MPEG-PS
File size                   : 73.3 MiB
Duration                    : 12 s 48 ms
Overall bit rate mode       : Constant
Overall bit rate            : 51.1 Mb/s
Frame rate                  : 25.000 FPS

Video
ID                          : 224 (0xE0)
Format                      : MPEG Video
Format version              : Version 2
Format profile              : Main@High
Format settings             : BVOP
Format settings, BVOP       : Yes
Format settings, Matrix     : Default
Format settings, picture st : Frame
Duration                    : 12 s 0 ms
Bit rate mode               : Constant
Bit rate                    : 50.0 Mb/s
Width                       : 1 920 pixels
Height                      : 1 080 pixels
Display aspect ratio        : 16:9
Frame rate                  : 25.000 FPS
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Interlaced
Scan order                  : Top Field First
Compression mode            : Lossy
Bits/(Pixel*Frame)          : 0.965
Time code of first frame    : 00:00:00:00
Time code source            : Group of pictures header
Stream size                 : 71.0 MiB (97%)

Audio
ID                          : 192 (0xC0)
Format                      : MPEG Audio
Format version              : Version 1
Format profile              : Layer 2
Duration                    : 12 s 48 ms
Bit rate mode               : Constant
Bit rate                    : 384 kb/s
Channel(s)                  : 2 channels
Sampling rate               : 48.0 kHz
Frame rate                  : 41.667 FPS (1152 SPF)
Compression mode            : Lossy
Stream size                 : 565 KiB (1%)
```

**Comment by Julusian** [comment]
Yes, a test video would be helpful. If you don't want to make the video public you can email it to me at git@julusian.co.uk

**Comment by Julusian** [comment]
Thanks for the files!

Fixed by 1f71ae64d3bee5df7f28f47fd6f8391f1c791fe7 released in v1.3.4

**Comment by 4l4R1** [comment]
Thanks, that was amazingly fast.
Couldn't resist and tested from home. Everything works.
